docker/release/docker-compose.yml

```diff
@@ -51,21 +69,49 @@ services:
       - POSTGRES_USER=timesketch
       - POSTGRES_PASSWORD=${POSTGRES_PASSWORD}
     restart: always
-    volumes:
-      - ./data/postgresql:/var/lib/postgresql/data
+    healthcheck:
+      test: ["CMD-SHELL", "bash -c '</dev/tcp/127.0.0.1/5432' || exit 1"]
+      interval: 30s
+      timeout: 5s
+      retries: 5
+    networks:
+      - timesketch_internal
+    volumes:https://github.com/cybersante
```

Comment:
Not sure what that github repo here means, could you clarify @lprat

Reply:
Sorry it's mistake! I fixed